lib/money/money.rb

```diff
@@ -312,6 +312,10 @@ def self.from_amount(amount, currency = default_currency, options = {})
     new(value, currency, options)
   end
 
+  class << self
+    alias from_cents new
```

Comment:
let's use alias_method, it is better understood and does not rely on lexical scoping